src/test/resources/org/jenkinsci/plugins/casc/integrations/global-matrix-auth/GlobalMatrixStrategy.yml

```diff
@@ -0,0 +1,23 @@
+jenkins:
+  systemMessage: "Welcome!"
+  numExecutors: 4
+  scmCheckoutRetryCount: 2
+  mode: NORMAL
+  scmCheckoutRetryCount: 4
+  securityRealm:
+    local:
+      allowsSignup: false
+      users:
+       - id: test
+         password: test
+
+  authorizationStrategy:
+    globalMatrix:
+      grantedPermissions:
```

Comment:
@ndeloof @ewelinawilkosz @MadsNielsen We need to agree on the format for permissions. https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/src/test/resources/org/jenkinsci/plugins/casc/integrations/role-strategy/RoleStrategy1.yml uses permission IDs while this plugin uses fully qualified names. Probably CasC plugin should offer a generic API for that